pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling<1.27.0"]
+requires = ["hatchling"]
```

Maybe add this to the metadata.

If someone will try to run `python -m build` with an older pkginfo, I expect they will get an error.

```suggestion
requires = ["hatchling", "pkginfo>=1.12.0"]
```

---

It actually builds fine without this &mdash; it is `twine` choking because of an old `pkginfo`.